### PR TITLE
Create task for adding students to slack section channels

### PR DESCRIPTION
**Pull request by samklq**
- Add unit test for task

### CODE REVIEW

File: student/management/commands/add_students_to_slack_section_channels.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    def handle(self, *args, **options):
+        batch_start_date = date.today() + timedelta(days=settings.DAYS_BEFORE_BATCH_FOR_ADDING_STUDENTS_TO_SECTION_CHANNELS)
+        batch_ids = Batch.objects.filter(start_date=batch_start_date).values_list('pk', flat=True)
+        section_ids = Enrolment.objects.filter(batch_id__in=batch_ids).values_list('section_id', flat=True)
+        slack_client = Slack()
+
+        for section_id in section_ids:
+            slack_section_channel_id = Section.objects.get(pk=section_id).slack_channel_id
+            slack_user_ids = list(StudentUser.objects.filter(pk__in=(Enrolment.objects.filter(section_id=section_id).values('student_user_id'))).values_list('slack_user_id', flat=True))
+            self.stdout.write(f"Adding users {slack_user_ids} to {slack_section_channel_id}")
```
Comment (rocketkai) [inline student/management/commands/add_students_to_slack_section_channels.py:23]: Nice logging

File: student/management/commands/add_students_to_slack_section_channels.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    def handle(self, *args, **options):
+        batch_start_date = date.today() + timedelta(days=settings.DAYS_BEFORE_BATCH_FOR_ADDING_STUDENTS_TO_SECTION_CHANNELS)
+        batch_ids = Batch.objects.filter(start_date=batch_start_date).values_list('pk', flat=True)
+        section_ids = Enrolment.objects.filter(batch_id__in=batch_ids).values_list('section_id', flat=True)
+        slack_client = Slack()
+
+        for section_id in section_ids:
+            slack_section_channel_id = Section.objects.get(pk=section_id).slack_channel_id
+            slack_user_ids = list(StudentUser.objects.filter(pk__in=(Enrolment.objects.filter(section_id=section_id).values('student_user_id'))).values_list('slack_user_id', flat=True))
```
Comment (rocketkai) [inline student/management/commands/add_students_to_slack_section_channels.py:22]: Should this line be decomposed?

**Review by rocketkai — APPROVED** [review]
LGTM

File: student/management/commands/add_students_to_slack_section_channels.py (hunk 2 above)
Comment (samklq) [reply student/management/commands/add_students_to_slack_section_channels.py:22]: can split up this way if you find it difficult to read

`student_user_ids = Enrolment.objects.filter(section_id=section_id).values('student_user_id')`
`slack_user_ids = list(StudentUser.objects.filter(pk__in=student_user_ids).values_list('slack_user_id', flat=True))

**Pull request merged**